app/rest/connector-generator/src/test/java/io/syndesis/connector/generator/swagger/TestHelper.java

```diff
@@ -56,4 +56,12 @@ private TestHelper() {
         return resource;
     }
 
+    /* default */ static String resource(final String path, final String alternative) throws IOException {
```

Comment:
I like this but how about expanding it even more, varargs loop through until find first resource? Slightly more flexible and might help us in future.

Reply:
Let's deal deal with that in the future, right now we need only one alternative (we check for JSON or YAML file present).